msg/RobotState.msg

```diff
@@ -0,0 +1,8 @@
+int8 ERROR = -1
+int8 E_STOP = 0
+int8 STANDBY = 1
+int8 DOCKING = 2
+int8 SUCCESS = 3
+
+int8 state_id
+string state_name
```

Comment:
The `state_name` is necessary? What is the reason?

Reply:
I had some idea, but right now, you are right, it's not necessary